axelrod/strategies/worseandworse.py

```diff
@@ -0,0 +1,35 @@
+from axelrod import Actions, Player
+from random import randint
+
+class WorseAndWorse (Player):
+    """
+    Defects with probability of 'current turn / total no. of turns'. Therefore
+    it is more and more likely to defect as the round goes on.
+
+    Names:
+        - worse_and_worse: [PRISON1998]
+
+    """
+
+    name = 'Worse and worse'
+    classifier = {
+        'memory_depth': float('inf'),
+        'stochastic': True,
+        'makes_use_of': set(['length']),
+        'long_run_time': False,
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def strategy(self, opponent):
+        current_round = len(self.history) + 1
+        expected_length = self.match_attributes['length']
+        try:
+            if randint(0, expected_length) < (current_round):
+                return Actions.D
+            return Actions.C
+        except:
```

Comment:
I haven't reviewed the strategy properly (once all tests are sorted I will) but this general except is not great: better to catch the actual error. Also, not sure why you're hardcoding 200, is that just for some tests when the `match_attributes` isn't passed? I think I'd prefer a different default behaviour there, perhaps just cooperate or defect or randomly cooperate or defect...